static/css/components/iaBar.less

```diff
@@ -7,8 +7,7 @@
   max-width: 960px;
   margin: 0 auto;
   -webkit-box-align: center;
-  -ms-flex-align: center;
-  align-items: center;
+  -ms-flex-align: center;  
```

Comment:
When using vendor prefixed properties, the standard one should always be applied.